.vscode/settings.json

```diff
@@ -22,6 +22,9 @@
 	"typescript.format.insertSpaceAfterTypeAssertion": true,
 	"typescript.format.insertSpaceBeforeFunctionParenthesis": true,
 	"typescript.format.semicolons": "insert",
+	"typescript.preferences.importModuleSpecifier": "relative",
+	"typescript.preferences.importModuleSpecifierEnding": "js",
```

Comment:
I had to add this to make TypeScript import the ghost `.js` files when using VSCode